### DISCUSSION TITLE
action-menu桌面模式用词歧义

### DISCUSSION THREAD

**Discussion by slime-butter** [vrcd-community/vrchat-docs-localization#59] [General]
## 原文
Desktop Mode
In desktop mode, press R to pull up the Action Menu. Move the mouse around to select items, and click. Click to back out of menus.

## 译文
桌面模式
在桌面模式下，按 `R键` 来弹出圆盘菜单。使用鼠标滑动并点击来进行选择。点击“返回上一级”选项可以返回上一级并退出当前菜单。

## 链接
https://docs.vrchat.com/docs/action-menu

## 歧义
返回上一级和退出当前菜单操作无法同时进行，需要一个新的对Click to back out of menus.的解释和翻译。

**Comment by yelandiealma** [comment]
点击“返回”选项可以返回上一级菜单。
這樣如何?按照當前簡中翻譯版，只有返回，我認為退出當前菜單的說法有點多餘
![螢幕擷取畫面 2024-01-26 111816](https://github.com/vrcd-community/vrchat-docs-localization/assets/97604181/b81d9147-c7b4-411a-b70a-6808768893c9)

**Reply by slime-butter** [reply]
好德！所以实际上Click to back out of menus需要被单独翻译为：单击R键以退出圆盘菜单
